cmd/cli/serve/ucan.go

```diff
@@ -133,6 +132,13 @@ func startServer(cmd *cobra.Command, _ []string) error {
 		return fmt.Errorf("loading config: %w", err)
 	}
 
+	if len(cfg.PDPServerURL) > 0 && cfg.ProofSet == 0 {
```

Comment:
what about this small change for consistency in the checks?
```suggestion
	if cfg.PDPServerURL != "" && cfg.ProofSet == 0 {
```